hudi-utilities/pom.xml

```diff
@@ -227,6 +227,10 @@
           <groupId>org.slf4j</groupId>
           <artifactId>slf4j-api</artifactId>
         </exclusion>
+        <exclusion>
+          <groupId>org.apache.hadoop</groupId>
+          <artifactId>hadoop-client</artifactId>
+        </exclusion>
```

Comment:
Could you clarify if this is needed?  Any implication on Spark bundle (e.g., missing Hadoop-related classes)?  Is this for test only?